### PR TITLE
ci: use travisci-tools script to trigger-fullstack-compat suite

### PR DESCRIPTION
**Pull request by yasirfolio3**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -35,12 +35,12 @@ jobs:
...
-        - "pip install awscli"
+      install: skip
       before_script:
-        - "aws s3 cp s3://optimizely-travisci-artifacts/ci/trigger_fullstack-sdk-compat.sh ci/ && chmod u+x ci/trigger_fullstack-sdk-compat.sh"
+        - mkdir $HOME/travisci-tools && pushd $HOME/travisci-tools && git init && git pull https://$CI_USER_TOKEN@github.com/optimizely/travisci-tools.git && popd
       script:
-        - "ci/trigger_fullstack-sdk-compat.sh"
+        - "$HOME/travisci-tools/fsc-trigger/trigger_fullstack-sdk-compat.sh"
       # travis_terminate not supported by darwin
       after_success: skip
```
Comment (juancarlostong) [inline .travis.yml]:
```suggestion
       after_success: travis_terminate 0
```
we can terminate now that we arent using darwin

File: .travis.yml [hunk 2]
```diff
@@ -35,12 +35,12 @@ jobs:
         - BUILD_NUMBER=$TRAVIS_BUILD_NUMBER
         - TESTAPP_TAG=master
       cache: false
+      language: minimal
```
Comment (juancarlostong) [inline .travis.yml:38]:
```suggestion
      language: minimal
      os: linux
```

File: .travis.yml [hunk 3]
```diff
@@ -35,12 +35,12 @@ jobs:
...
-      install:
-        - "pip install awscli"
+      install: skip
       before_script:
-        - "aws s3 cp s3://optimizely-travisci-artifacts/ci/trigger_fullstack-sdk-compat.sh ci/ && chmod u+x ci/trigger_fullstack-sdk-compat.sh"
+        - mkdir $HOME/travisci-tools && pushd $HOME/travisci-tools && git init && git pull https://$CI_USER_TOKEN@github.com/optimizely/travisci-tools.git && popd
       script:
-        - "ci/trigger_fullstack-sdk-compat.sh"
+        - "$HOME/travisci-tools/fsc-trigger/trigger_fullstack-sdk-compat.sh"
       # travis_terminate not supported by darwin
```
Comment (juancarlostong) [inline .travis.yml]:
```suggestion

```

**Review by juancarlostong — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (yasirfolio3) [reply .travis.yml:38]: Changes made.

**Review by juancarlostong — APPROVED** [review]

**Pull request merged**